nucleotide_transformer/sCellTransformer/layers.py

```diff
@@ -0,0 +1,424 @@
+import logging
```

Comment:
Can we re-use the layers from https://github.com/instadeepai/nucleotide-transformer/blob/main/nucleotide_transformer/layers.py?